lib/msf/core/rpc/v10/rpc_module.rb

```diff
@@ -247,7 +247,7 @@ def rpc_info(mtype, mname)
     if m.type == 'auxiliary' || m.type == 'post'
       res['actions'] = {}
       m.actions.each_index do |i|
-        res['actions'][i] = m.actions[i].name
+        res['actions'][m.actions[i].name] = m.actions[i].description
```

Comment:
Looks like this is a backwards incompatible change that might break existing integrations 👀

Reply:
This will require some views and/or RPC classes to be updated in Pro.

Reply:
If we circle back around to this, it might make more sense to make this an array of objects so we can add additional properties in the future :+1: